README.md

```diff
@@ -91,7 +91,7 @@ public class WeatherIT {
     @Test
     public void checkWeather() {
         graphql.execute("{ weather { temperature } }", (header, body) -> {
-            body.get("weather.temperature").shouldBe(lessThan(100));
+            body.get("data.weather.temperature").shouldBe(lessThan(100));
```

Comment:
is `data.` optional here?

Reply:
No, it exists in the response so when interacting with the body DataNode directly you need it.  The optionality of it is implemented in the resolution customisation when we convert the groovy closure into the relevant validator interface.